src/goSuggest.ts

```diff
@@ -44,6 +44,14 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 		return this.provideCompletionItemsInternal(document, position, token, vscode.workspace.getConfiguration('go'));
 	}
 
+	private isInBlockComment(document: vscode.TextDocument, position: vscode.Position): boolean {
+		let text = document.getText();
+		let currentOffset = document.offsetAt(position);
+		let lastBlockCommentStartIndex = text.lastIndexOf('/*', currentOffset);
+		let lastBlockCommentEndIndex = text.lastIndexOf('*/', currentOffset);
+		return lastBlockCommentStartIndex > lastBlockCommentEndIndex;
```

Comment:
There are a few cases where this will give a false positive
One example is
```
	hi := "/*"

	bye := "*/"
```
You won't get completions between the above 2 lines

OR in between the below 2 lines
```
hi := "/*" 
	
	// This is a comment */
```

Reply:
Hi @ramya-rao-a, I've created a separate PR here (https://github.com/ramya-rao-a/go-outline/pull/7) to push this fix forward.